router.go

```diff
@@ -43,13 +48,13 @@ func assembleMiddleware() {
 }
 
 func assembleRoutes(build, hash string) {
-	buildstamp = buildstamp
```

Comment:
```go vet && golint``` picked this up. I notice the ```buildstamp``` is set here and also above in ```NewRouter()```. Not sure if this is intentional?